src/resources/views/crud/columns/inc/wrapper_end.blade.php

```diff
@@ -1 +1,9 @@
-</{{$column['wrapper']['element'] ?? 'a'}}>
\ No newline at end of file
+@php
+    // define the wrapper element
+    $wrapperElement = isset($column['wrapper']['element']) ? $column['wrapper']['element'] : 'a';
```

Comment:
```suggestion
    $wrapperElement = $column['wrapper']['element'] ?? 'a';
```

Reply:
This wouldn't work, since `??` doesn't check if the array key exists.